glue_vispy_viewers/common/vispy_data_viewer.py

```diff
@@ -28,8 +31,12 @@ def __init__(self, session, parent=None):
 
         toolbar = self._toolbar_cls(vispy_widget=self._vispy_widget, parent=self)
         self.addToolBar(toolbar)
+        # try customize tool
+        # super(BaseVispyViewer, self).initialize_toolbar()
+
 
         self.status_label = None
+        self.client = None  # don't know what to implement here
```

Comment:
As our viewer doesn't use `client`, but it's needed for this test code, so here I assign `None` to client but still get following error, do you know how to solve this? @astrofrog  Thanks!

> def restore_layers(self, rec, context):
        """
            Given a list of glue-serialized layers, restore them
            to the viewer
            """
        # if this viewer manages a client, rely on it to restore layers
        if hasattr(self, 'client'): 
              return self.client.restore_layers(rec, context)
              **AttributeError: 'NoneType' object has no attribute 'restore_layers'**